src/backend/config/config.local.json.base

```diff
@@ -2,5 +2,9 @@
   "googlecalendar": {
```

Comment:
Kan vi lyfta in hela denna filen i config.js och därifrån mappa mot environment vars?